### PR TITLE
Add documentation for using "cloud" kernels

### PR DESCRIPTION
**Pull request by jxmx**

### CODE REVIEW

File: docs/basic/cloud-kernel.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+# Using AllStarLink/app_rpt with Cloud Kernels
```
Comment (Allan-N) [inline docs/basic/cloud-kernel.md:1]: I'm thinking that this entire page should be moved to docs/adv-topics

File: docs/basic/cloud-kernel.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+Many virtual hosts or VPS providers leverage the Debian "cloud kernel"
+version of the Debian Linux OS. These providers include Amazon AWS,
+Microsoft Azure, Linode, Vultr, and DigitalOcean. Using their stock
```
Comment (Allan-N) [inline docs/basic/cloud-kernel.md:5]: The most recent VPS provider reported to have issues was DreamHost.  Do we want to expand the list?

File: docs/user-guide/install.md [hunk 3]
```diff
@@ -14,10 +14,8 @@ installation on Raspberry Pis, consider using the [AllStarLink 3 Pi Appliance](p
 
     1. When the VPS provider is using a customized Debian using the "cloud" kernel. If the
     output of `uname -r` refers to the kernel as something similar to 
-    "6.1.0-12-cloud-x86_64" then you **cannot** use that OS for ASL because it lacks the
-    capabilities build the needed kernel modules. You must install a standard Debian 12
-    image which is probably a "custom install" for your provider. Ask your VPS provider
-    for support.
+    "6.1.0-12-cloud-x86_64" then you must [follow this procedure](/basic/cloud-kernel)
+    prior to installing ASLv3.
```
Comment (Allan-N) [inline docs/user-guide/install.md:18]: Do we want to expand somewhere in this doc about the ASL3 behavior that might be observed if you don't have a kernel that plays nicely?

File: mkdocs.yml [hunk 4]
```diff
@@ -73,6 +73,7 @@ nav:
     - user-guide/passwords.md
     - user-guide/support.md
     - user-guide/betatest.md
+    - basic/cloud-kernel.md
```
Comment (Allan-N) [inline mkdocs.yml:76]: Move to adv-topics?

File: docs/basic/cloud-kernel.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+	depmod...
+	```
+	
+* Load the dahdi module:
+
+	```
+	modprobe dahdi
+	```
+	
+* Check that the module is loaded with `1smod | grep dahdi`:
```
Comment (Allan-N) [inline docs/basic/cloud-kernel.md:117]: 1smod --> lsmod

**Review by Allan-N — CHANGES_REQUESTED** [review]

File: docs/basic/cloud-kernel.md (hunk 1 above)
Comment (jxmx) [reply docs/basic/cloud-kernel.md]: Done

File: mkdocs.yml (hunk 4 above)
Comment (jxmx) [reply mkdocs.yml:76]: ok

Comment (Allan-N) [reply mkdocs.yml:76]: Looks like you moved the file so this reference needs to be updated to match.

**Review by Allan-N — CHANGES_REQUESTED** [review]

**Review by Allan-N — APPROVED** [review]
Looks good!

**Pull request merged**